org.kde.dragonplayer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "org.kde.dragonplayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
```

Comment:
should be bumped no?

Reply:
That's the latest KDE Runtime that we have (based on fdo 22.08)